cloudant-client/src/test/java/com/cloudant/tests/UnicodeTest.java

```diff
@@ -70,13 +54,10 @@
     private static final String TESTSTRING = "Gr\u00fc\u00dfe \u65e5\u672c\u8a9e \uD834\uDD1E.";
     private static final String TESTSTRING_ESCAPED = "Gr\\u00fc\\u00dfe \\u65e5\\u672c\\u8a9e " +
             "\\uD834\\uDD1E.";
-
-    private Database db;
-
-    @Before
-    public void setup() {
-        db = dbResource.get();
-    }
+    private static final String TESTJSON = "{\"_id\":\"literal\"," +
+            "\"_rev\":\"1-39933759c7250133b6039d94ea09134f\",\"foo\":\"Grüße 日本語 \uD834\uDD1E.\"}\n";
+    private static final String TESTJSON_ESCAPED = "{\"_id\":\"escaped\"," +
```

Comment:
Shouldn't this have `\\` similar to TESTSTRING_ESCAPED above?

Reply:
Thanks for spotting this inconsistency. These are actually used as "expected" values for assertions later, and the json deserialiser will always unescape these values, so they are correct in that regard.

However, the logic and naming of some variables is confusing so I've fixed that in e8a6acb